code_to_gast/helpers/js_helpers.py

```diff
@@ -25,41 +25,23 @@ def program_to_gast(node):
 """
 converts a python ast BinOp and converts it to a gast node
 """
-def binOp(bop):
+def binOp_to_gast(bop):
```

Comment:
`bop` --> `binOp_node` or probably better to go with `bin_op_node` all snake case while you're here.

Reply:
Resolved.